SPECS/libvirt/libvirt.spec

```diff
@@ -1,654 +1,1970 @@
-Requires:       iproute
-# for modprobe of pci devices
-Requires:       module-init-tools
-Requires:       numad
-Requires:       polkit >= 0.112
-Requires(post): systemd-sysv
-# For service management
-Requires(post): systemd-units
-Requires(postun): systemd-units
-# For uid creation during pre
-Requires(pre):  shadow-utils
-Requires(preun): systemd-units
-%ifarch x86_64
-# For virConnectGetSysinfo
-Requires:       dmidecode
+%if %{with_mingw32}
+BuildRequires: mingw32-filesystem
+BuildRequires: mingw32-gcc
+BuildRequires: mingw32-binutils
+BuildRequires: mingw32-glib2 >= 2.48
+BuildRequires: mingw32-gnutls
+BuildRequires: mingw32-gettext
+BuildRequires: mingw32-libxml2
+BuildRequires: mingw32-portablexdr
+BuildRequires: mingw32-dlfcn
+BuildRequires: mingw32-libssh2
+BuildRequires: mingw32-curl
 %endif
+%if %{with_mingw64}
+BuildRequires: mingw64-filesystem
+BuildRequires: mingw64-gcc
+BuildRequires: mingw64-binutils
+BuildRequires: mingw64-glib2 >= 2.48
+BuildRequires: mingw64-gnutls
+BuildRequires: mingw64-gettext
+BuildRequires: mingw64-libxml2
+BuildRequires: mingw64-portablexdr
+BuildRequires: mingw64-dlfcn
+BuildRequires: mingw64-libssh2
+BuildRequires: mingw64-curl
+%endif
+
+%description
+Libvirt is a C toolkit to interact with the virtualization capabilities
+of recent versions of Linux (and other OSes). The main package includes
+the libvirtd server exporting the virtualization support.
+
+%if %{with_native}
+%package docs
+Summary: API reference and website documentation
+
+%description docs
+Includes the API reference for the libvirt C library, and a complete
+copy of the libvirt.org website documentation.
+
+%package daemon
+Summary: Server side daemon and supporting files for libvirt library
+
+# All runtime requirements for the libvirt package (runtime requrements
+# for subpackages are listed later in those subpackages)
+
+# The client side, i.e. shared libs are in a subpackage
+Requires: libvirt-libs = %{version}-%{release}
+Requires: libvirt-daemon-common = %{version}-%{release}
+Requires: libvirt-daemon-lock = %{version}-%{release}
+Requires: libvirt-daemon-plugin-lockd = %{version}-%{release}
+Requires: libvirt-daemon-log = %{version}-%{release}
+Requires: libvirt-daemon-proxy = %{version}-%{release}
 
 %description daemon
 Server side daemon required to manage the virtualization capabilities
 of recent versions of Linux. Requires a hypervisor specific sub-RPM
 for specific drivers.
 
+%package daemon-common
+Summary: Files and utilities used by daemons
+Requires: libvirt-libs = %{version}-%{release}
+# The libvirt-guests.sh script requires virsh from libvirt-client subpackage,
+# but not every deployment wants to use libvirt-guests service. Using
+# Recommends here will install libvirt-client by default (if available), but
+# RPM won't complain if the package is unavailable, masked, or removed later.
+Recommends: libvirt-client = %{version}-%{release}
+# for /sbin/ip
+Requires: iproute
+%if 0%{!?mariner}
```

Comment:
this is a new dependency that i don't see in mariner yet.  i commented it out for now.  not sure what the impact is of that, but the buddy build pipeline succeeds.

Reply:
iproute-tc looks like part of new iproute (https://[src.fedoraproject.org/rpms/iproute/raw/main/f/iproute.spec](https://src.fedoraproject.org/rpms/iproute/raw/main/f/iproute.spec).  

can uncomment this when iproute is upgraded.